scripts/rnnlm/initialize_matrix.pl

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env perl
+# Copyright 2017     Hossein Hadian
+
+$randseed = 0;
+$shifted = 0;
+
+do {
+  $shifted=0;
+  if ($ARGV[0] eq "--first-column") {
```

Comment:
Thank you Dr. Povey. Added --first-element option, replaced all invocations of the python script with invocations of the perl script, and the python script is deleted.